src/pages/Files.vue

```diff
@@ -88,59 +95,54 @@ export default {
     </oc-application-menu>
 
-    <oc-topbar variation="secondary">
-      <template slot="left">
-        <div class="uk-navbar-item uk-logo">
-          <oc-icon name="home"></oc-icon>
-        </div>
-        <oc-breadcrumb :items="[{text:'Home',to:'/'},{text:'Folder',to:{path:'folder'}},{text:'Subfolder'}]"></oc-breadcrumb>
-      </template>
-        <template slot="title">
-            <div class="uk-navbar-item">
-                <oc-search-bar label="Search"></oc-search-bar>
+    <oc-app-layout :rightHidden="rightHidden">
+
+      <template slot="center">
+        <section class="uk-background-muted uk-padding-small">
+          <div uk-grid class="uk-grid-small uk-flex-middle">
+            <div class="uk-width-1-1 uk-width-auto@l">
+              <oc-breadcrumb first-is-home :items="[{text:'Folder',to:{path:'folder'}},{text:'Subfolder'}]"></oc-breadcrumb>
             </div>
-        </template>
-        <template slot="right">
-            <div class="uk-navbar-item">
-                <oc-menu buttonText="+ New">
-                  <template slot="subnav">
-                    <oc-menu-item icon="create_new_folder" content="New folder …" />
-                    <oc-menu-item icon="save" content="New file …" />
-                  </template>
-                </oc-menu>
+            <div class="uk-width-expand">
+              <div class="uk-align-right">
+                <oc-search-bar label="Search"></oc-search-bar>
+              </div>
             </div>
-            <div class="uk-navbar-item">
-                <span>3 results</span>
+            <div class="uk-width-auto uk-visible@l">
+              <oc-menu buttonText="+ New">
+                <template slot="subnav">
+                  <oc-menu-item icon="create_new_folder" content="New folder …" />
+                  <oc-menu-item icon="save" content="New file …" />
+                </template>
+            </oc-menu>
             </div>
-            <div class="uk-navbar-item">
-                <oc-menu>
-                    <template slot="activator">
-                        <div>
-                            <oc-icon name="filter_list"></oc-icon>
-                        </div>
-                    </template>
-                    <template slot="subnav">
-                      <oc-menu-item>
-                        Files<oc-checkbox></oc-checkbox>
-                      </oc-menu-item>
-                      <oc-menu-item>
-                        Folders<oc-checkbox></oc-checkbox>
-                      </oc-menu-item>
-                      <oc-menu-item>
-                        Hidden files<oc-checkbox></oc-checkbox>
-                      </oc-menu-item>
-                      <oc-menu-item>
-                        Files by name<oc-text-input></oc-text-input>
-                      </oc-menu-item>
-                    </template>
-                </oc-menu>
+            <div class="uk-visible@l">
```

Comment:
the filter menu will be a component of it's own - see the proposal if @LukasHirt in #197

Reply:
I removed it so we can plug in the component later.